pages/1.11/release-notes/1.11.2/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.2
+title: Release Notes for 1.11.2
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.2
+---
+
+These are the release notes for DC/OS 1.11.2.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.2 includes the following:
+
+- Apache Mesos 1.5.1-dev [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
+- Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# Issues Fixed in DC/OS 1.11.2
+
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file.
+- DCOS-20514 - Added licensing information to the diagnostics bundle.
+- DCOS-21596 - Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS_OSS-2360 - DC/OS Metrics: metric names are sanitized for better compatibility with Prometheus.
+- DCOS_OSS-2378 - DC/OS Net: Improved stability of distribution protocol over TLS. [enterprise type="inline" size="small" /]
+- DC/OS UI: Incorporated [multiple](https://github.com/dcos/dcos/pull/2799) fixes and improvements.
+
+
+# Notable Changes in DC/OS 1.11.2
+
+- DCOS-21958 - Admin Router on the DC/OS master nodes now does not support TLS 1.1 and the 3DES bulk encryption algorithm anymore by default.
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /] 
+- DCOS-29122 - Support for launching services in a remote region.
```

Comment:
I am requesting @benclarkwood  and @gabrielhartmann  to review these entries for DC/OS 1.11.2 changelog.

```
+- DCOS-29122 - Support for launching services in a remote region.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
```

* Are these accurate descriptions for the feature supported in 1.11.2?
* Is this accomplished in 1.11.2?
* Should this be mentioned in changelog? (A noteworthy change that a customer is going to find it useful). 

I see an epic called (Region Aware Scheduling in Data Services https://jira.mesosphere.com/browse/DCOS-29023) still in In-progress, so I am unable to take call on stating these two entries as "features" supported by DC/OS 1.11.2 

TIA!

Reply:
@orsenthil EPIC https://jira.mesosphere.com/browse/DCOS-29575 was in progress because the documentation ticket is open. Sukesh should close this ticket. We have released this and you can check the service docs - https://docs.mesosphere.com/services/kafka/2.2.0-1.0.0/release-notes/ This feature is released after 1.11 release so to answer your questions - 

1. Are these accurate descriptions for the feature supported in 1.11.2? 
Yes; Check this release notes https://docs.mesosphere.com/services/kafka/2.2.0-1.0.0/release-notes/ 
2. Is this accomplished in 1.11.2? 
Yes
3. Should this be mentioned in changelog? (A noteworthy change that a customer is going to find it useful).
Yes

Reply:
Thank you! @imaxxs !